index.js

```diff
@@ -113,6 +112,10 @@ var BrowserStackBrowser = function(id, emitter, args, logger,
              process.env.BUILD_TAG || process.env.CIRCLE_BUILD_NUM || null
     };
 
+    if (args.real_mobile) {
```

Comment:
Good point. I decided to move in the direction of "if set", which covers us if BrowserStack starts allowing booleans.